### ISSUE TITLE
Does this support spike robots?

### ISSUE THREAD

**Issue by Luxvao** [ev3dev/ev3dev#1560]
**Question**

It would be really cool if this supported spike robots, but it has different ports so idk.

**Robot (please complete the following information):**
 - Device: Spike

**Comment by dlech** [comment]
No, ev3dev requires Linux. The SPIKE hubs are not powerful enough to run Linux. You may be interested in https://pybricks.com instead.

**Comment by Luxvao** [comment]
@dlech Is there a way I could program spike robots with Rust, since it's my fav language?

**Comment by dlech** [comment]
You would need to create rust bindings around the PBIO library from Pybricks.
